### PR TITLE
Speed up dependabot enabling script

### PR DESCRIPTION
**Pull request by 0atman**
Ready for cronning

I've added a `DRY_RUN` env option that doesn't make any PUTs:
```javascript
DRY_RUN=true GITHUB_ORG=alphagov TOKEN=yourtoken aws-vault exec [yourrole] -- docker-compose run dashboard python cyber_dependabot.py
```

### CODE REVIEW

**Comment by 0atman** [comment]
An example (just 10 repos) of the output from this is:

```python
python cyber_dependabot.py 
INFO:root:Starting processing 10 repos...
INFO:root:repo: publisher, PUT: 204, 
INFO:root:repo: gds-sso, PUT: 204, 
INFO:root:repo: whitehall, PUT: 204, 
INFO:root:repo: capistrano_rsync_with_remote_cache, PUT: 204, 
INFO:root:repo: frontend, PUT: 204, 
INFO:root:repo: slimmer, PUT: 204, 
INFO:root:repo: govspeak, PUT: 204, 
INFO:root:repo: imminence, PUT: 204, 
INFO:root:repo: plek, PUT: 204, 
INFO:root:repo: static, PUT: 204, 
INFO:root:Counter({204: 10})
```

**Comment by 0atman** [comment]
By my estimates (above), it should take less than 3 minutes to run this.

**Comment by 0atman** [comment]
https://github.com/alphagov/cyber-security-security-advisory-dashboard/pull/106/commits/a1ccc7c54450cef69c57a708b04e7c83c1c3f51c is juuuuust for @akinnane :ok_hand:

**Comment by akinnane** [comment]
randy_approves
LGTM

File: cyber_dependabot.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        return 200
+    else:
+        r = put(f"/repos/{repo.owner.login}/{repo.name}/vulnerability-alerts")
+        logging.info(f"repo: {repo.name}, PUT: {r.status_code}, {r.text}")
+        return r.status_code
+
+
+def enable_vulnerability_alerts() -> None:
+    """
+    Enables vulnerability alerts on every repo in repositories.json.
```
Comment (alice-carr) [inline cyber_dependabot.py]: maybe i'm missing something but how are we putting new repos in repositories.json?

File: cyber_dependabot.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        return p.map(f, collection)
+
+
+def put(path: str) -> requests.models.Response:
+    """
+    Adapter of `requests.put()`, supplying github credentials.
+    """
+    ROOT_URL = "https://api.github.com"
+    headers = {
+        "Authorization": f'token {os.environ.get("TOKEN", "")}',
```
Comment (alice-carr) [inline cyber_dependabot.py]: we're sending the github token to the lambda as an encrypted kms token that will need to be decrypted in the python function itself - might be worth putting this in now?
i.e.:
ENCRYPTED_GITHUB_TOKEN = os.environ.get["GITHUB_TOKEN"]
DECRYPTED_GITHUB_TOKEN = boto3.client('kms').decrypt(CiphertextBlob=b64decode(ENCRYPTED_GITHUB_TOKEN))['Plaintext']

File: cyber_dependabot.py (hunk 1 above)
Comment (0atman) [reply cyber_dependabot.py]: great question, it's created by the audit

**Pull request merged**